app/controllers/api/v1/moves_controller.rb

```diff
@@ -98,7 +99,7 @@ def import_moves_from_nomis(move: nil)
       def from_locations
         @from_locations ||=
           if filter_params[:from_location_id]
-            [Location.find(filter_params[:from_location_id])]
+            filter_params[:from_location_id].collect { |id| Location.find(id) }
```

Comment:
Could this be `Location.where(id: filter_params[:from_location_id])` to reduce the number of database queries?